examples/sn-react-component-docs/.storybook/docgen-plugin.js

```diff
@@ -0,0 +1,51 @@
+const docGen = require('react-docgen-typescript')
+const docGenLoader = require('react-docgen-typescript-loader/dist/generateDocgenCodeBlock.js')
+const ts = require('typescript')
+
+class DocgenPlugin {
+  apply(compiler) {
+    compiler.hooks.compilation.tap('DocgenPlugin', compilation => {
+      compilation.hooks.seal.tap('DocgenPlugin', modules => {
+        const modulesToProcess = []
+        compilation.modules.forEach(module => {
+          // Skip ignored / external modules
+          if (!module.built || module.external || !module.rawRequest) {
+            return
+          }
+          if (/\/src.+.tsx/.test(module.request)) {
```

Comment:
This regexp is not work on windows.
```js
/\/src.+.tsx/.test(module.request)
```
windows separator is \
this code work for me on windows
```diff
    const ts = require("typescript");
+   const path = require("path");

    class DocgenPlugin {
      apply(compiler) {
+       const re = RegExp(`\\${path.sep}src.+\.tsx`);
        compiler.hooks.compilation.tap("DocgenPlugin", compilation => {
          compilation.hooks.seal.tap("DocgenPlugin", modules => {
            const modulesToProcess = [];
            compilation.modules.forEach(module => {
              const { built, external, rawRequest, request } = module;
              // Skip ignored / external modules
              if (!module.built || module.external || !module.rawRequest) {
                return;
              }
-             if (/\/src.+.tsx/.test(module.request)) {
+             if (re.test(module.request)) {
                modulesToProcess.push(module);
              }
            });
```

Reply:
You are right. I've never realized that. I use wsl in windows for development and our builds are on a linux machine. Thanks for pointing that out :)